src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java

```diff
@@ -503,6 +508,51 @@ private void trackVideo(String eventName, TrackPayload track) {
       case "Video Playback Seek Completed":
         heartbeat.trackEvent(MediaHeartbeat.Event.SeekComplete, null, null);
         break;
+
+      case "Video Ad Break Started":
+        Properties videoAdBreakProperties = track.properties();
+        MediaObject mediaAdBreakInfo =
+            MediaHeartbeat.createAdBreakObject(
+                videoAdBreakProperties.getString("title"),
+                videoAdBreakProperties.getLong("indexPosition", 1), // Segment does not spec this
+                videoAdBreakProperties.getDouble("startTime", 0));
+
+        heartbeat.trackEvent(MediaHeartbeat.Event.AdBreakStart, mediaAdBreakInfo, null);
+        break;
+
+      case "Video Ad Break Completed":
+        heartbeat.trackEvent(MediaHeartbeat.Event.AdBreakComplete, null, null);
+        break;
+
+      case "Video Ad Started":
+        Properties videoAdProperties = track.properties();
+        Map<String, String> standardAdMetadata = new HashMap<>();
+        Properties adProperties = mapStandardVideoMetadata(videoAdProperties, standardAdMetadata);
+        HashMap<String, String> adMetadata = new HashMap<>();
+        adMetadata.putAll(adProperties.toStringMap());
+
+        MediaObject mediaAdInfo =
+            MediaHeartbeat.createAdObject(
+                videoAdProperties.getString("title"),
```

Comment:
Feels like we should have some fixtures without all of these properties.

Reply:
Gotcha, keeping notes of all these and will implement in future PRs.